RecoEgamma/EgammaIsolationAlgos/src/EgammaRecHitIsolation.cc

```diff
@@ -96,6 +98,13 @@ double EgammaRecHitIsolation::getSum_(const reco::Candidate* emObject, bool retu
           float phiDiff = reco::deltaPhi(phi, phiclus);
           float energy = j->energy();
 
+          float rhThres = 0.0;
+          if (thresholds != nullptr) {
+            rhThres = (*thresholds)[j->detid()];  // access ECAL PFRechit thresholds for noise cleaning
+          }
```

Comment:
```suggestion
          float rhThres = (thresholds != nullptr) ? (*thresholds)[j->detid()] : 0.f;
```

is more compact (of course depending on what you decide about https://github.com/cms-sw/cmssw/pull/43535/files#r1423888388)